load_data.py

```diff
@@ -65,17 +79,22 @@ def data_loader(dataset, file_name, dtype):
         with open(str(DATA_DIR + '/cifar-10-batches-py/' + file_name), 'rb') as fo:
             data_dict = cPickle.load(fo)
 
-        data = preprocess_cifar_imgs_(np.asarray(data_dict['data'], dtype=dtype))
+        data = preprocess_cifar_imgs_(np.asarray(data_dict['data'], dtype=dtype),
+                                      dtype=dtype,
```

Comment:
Is this ```dtype=dtype``` necessary?

Reply:
Yes, it is necessary for converting to grayscale